rdpy/core/type.py

```diff
@@ -795,7 +796,11 @@ def __write__(self, s):
             toWrite += self._until
             
         if self._unicode:
-            s.write(encodeUnicode(self.value))
+            val = self.value
+            encVal = encodeUnicode(self.value)
+            log.debug(encVal)
```

Comment:
Does these mean the library now writes out to a log every encoded string it ever encounters?